modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -87,12 +87,12 @@
 
 /obj/item/toy/plush/mammal/fox
 	name = "Fox plushie"
-	desc = "An adorable stuffed toy of a SecHound."
+	desc = "An adorable stuffed toy of an Fox."
```

Comment:
"a fox"

Reply:
done . thank god you're here ;_; . also I had an unneeded space in the borb part . fixed it too.